apps/studio/components/interfaces/QueryPerformance/QueryPerformanceGrid.tsx

```diff
@@ -31,6 +32,11 @@ interface QueryPerformanceGridProps {
   queryPerformanceQuery: DbQueryHook<any>
 }
 
+// Load the monaco editor client-side only (does not behave well server-side)
+const Editor = dynamic(() => import('@monaco-editor/react').then(({ Editor }) => Editor), {
```

Comment:
@taishikato are we doing this anywhere else? (or everywhere else?)

Reply:
gracias!